pkg/services/store/kind/geojson/summary.go

```diff
@@ -0,0 +1,71 @@
+package geojson
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/grafana/grafana/pkg/models"
+)
+
+func GetObjectKindInfo() models.ObjectKindInfo {
```

Comment:
I could make a new PR for this... but seems ok here as we are still fleshing out the kind registry